src/workloads/scale/InsertBigDocs.yml

```diff
@@ -0,0 +1,21 @@
+# This workload loads 500 rows into a collection, each of which is approximately 15MB in size.
+
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/replication"
+
+Actors:
+
+- Name: Loader
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &DB test
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 500
+    BatchSize: 1
+    Document:
+      x: {^RandomInt: {min: 0, max: 2147483647}}
+      string0: {^RandomString: {length: 15000000}}
+  - &Nop {Nop: true}
```

Comment:
You don't need this `- &Nop {Nop: true}` line since there is only one Actor in the worklodad